### PR TITLE
Added Feature to consume Leaderboard api

### PR DESCRIPTION
**Pull request by r-ahmed2022**
In this pull request, I completed the following:-
- connect to Leaderboard API 
- Create a new game with the name of your choice by using the API.
- Implement the "Refresh" button 
- Implement the form "Submit" button (sending data to the API).
- Use arrow functions instead of the function keyword.
- Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/modules/showTopPlayers.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    } else if (index === 1) {
+      document.querySelector('.second-person').innerHTML = `${play.user}`;
+      document.querySelector('.second-score').innerHTML = `${play.score}`;
+    } else if (index === 2) {
+      document.querySelector('.third-person').innerHTML = `${play.user}`;
+      document.querySelector('.third-score').innerHTML = `${play.score}`;
+    } else if (index > 2 && index < 10) { output.innerHTML += `<p class="player">${play.user} : ${play.score}</p>`; }
+  });
+};
+export default showTopPlayers;
```
Comment (zieeco) [inline src/modules/showTopPlayers.js]: - When a user adds a score and name, it does nothing. please fix this.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @r-ahmed2022 


Great work  so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/showTopPlayers.js (hunk 1 above)
Comment (r-ahmed2022) [reply src/modules/showTopPlayers.js]: Done as advised

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @r-ahmed2022 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**